classes/Infrastructure/Event/TwigGlobalsListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
     public static function getSubscribedEvents()
     {
         return [
-            KernelEvents::REQUEST => ['onKernelRequest', 256],
+            KernelEvents::REQUEST => ['onKernelRequest', 128],
```

Comment:
The priority had to be lowered because this listener would otherwise start a session in integration tests before Silex' test session listener is called.

Reply:
Thank you for clarifying, @derrabus!